### PR TITLE
refactor: `Formatter.scala` to be consistent with `Console.flix`

### LINKED ISSUE #4459: Refactor `Formatter.scala` to be consistent with `Console.flix`
- Offer the same set of color functions, e.g. `red` and `bgRed`, etc.
- Use the same colors in Scala as in Flix.

### PR DESCRIPTION
**Pull request by ninaandrup**
Closes #4459

### CODE REVIEW

File: main/src/ca/uwaterloo/flix/util/Formatter.scala [hunk 1]
```diff
@@ -131,33 +208,71 @@ object Formatter {
 
     override def black(s: String): String = Console.BLACK + s + Console.RESET
 
+    override def brightBlack(s: String): String = fgColor(128, 128, 128, s)
```
Comment (ninaandrup) [inline main/src/ca/uwaterloo/flix/util/Formatter.scala:211]: @magnus-madsen  Should the colors be added to AnsiColor.scala instead of using rgb?

Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/util/Formatter.scala:211]: What do you mean?

Comment (ninaandrup) [reply main/src/ca/uwaterloo/flix/util/Formatter.scala:211]: Like for black, where I can write:
```
  override def black(s: String): String = Console.BLACK + s + Console.RESET
```
The `Console.BLACK` part.

Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/util/Formatter.scala:211]: Oh. I would avoid using Console and just use rgb-- for consistency.

Comment (ninaandrup) [reply main/src/ca/uwaterloo/flix/util/Formatter.scala:211]: Should I change the ones using `Console` to `rgb`? Or just leave them as is

Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/util/Formatter.scala:211]: Change them to make it consistent

**Comment by magnus-madsen** [comment]
Thanks!

**Pull request merged**